frontend/packages/ceph-storage-plugin/src/components/ocs-install/ocs-install.scss

```diff
@@ -47,3 +47,14 @@
   display: flex;
   justify-content: space-between;
 }
+
+// Restrict node-list widths
+.ceph-ocs-install__node-list {
+  @media (min-width: 769px) {
+    max-width: 100% !important;
```

Comment:
Is it not working without `!important`? It's not a good practice to use it.

Reply:
We need to override some media queries in `co-m-pane__form`, so we need to use `!important`

Reply:
Ahh is it marked important? Then only we require important over here otherwise internal class always has priority.

Reply:
yes, but since we are using both classes here as internal class https://github.com/openshift/console/pull/6439/files#diff-10211f95caf12c0c4e086214ec83e140R120, thats why used `!important`.